baseplate/frameworks/queue_producer/avro_kafka.py

```diff
@@ -0,0 +1,97 @@
+import logging
```

Comment:
it'd mirror the existing structure if this were in baseplate/clients/kafka.py (see https://github.com/reddit/baseplate.py/blob/master/baseplate/clients/kombu.py)